pydotorg/urls.py

```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```

Comment:
I totally get that and understand why it's happened (@tjguk is a Windows user so the line endings are probably different). Just ensuring that *the important change* wasn't lost in the mega-diff. ;-)